app/controllers/api/providers_controller.rb

```diff
@@ -86,6 +86,13 @@ def import_vm_resource(type, id = nil, data = {})
       end
     end
 
+    def options
+      all_options = ManageIQ::Providers::BaseManager.leaf_subclasses.inject({}) do |ao, ems|
+        ao.merge(ems.ems_type => ems.respond_to?(:options_description) ? ems.options_description : {})
```

Comment:
Changed, tests here will only be green after https://github.com/ManageIQ/manageiq/pull/15799 is merged